docs/resource/process.md

```diff
@@ -156,13 +156,19 @@ Examples for some JavaScript runtimes
 
 ### .NET Runtimes
 
-TODO(<https://github.com/open-telemetry/opentelemetry-dotnet/issues/1281>): Confirm the contents here
+.NET instrumentation should fill in the values by following values:
 
-| Value | Description |
-| --- | --- |
-| `dotnet-core` | .NET Core, .NET 5+ |
-| `dotnet-framework` | .NET Framework |
-| `mono` | Mono |
+- `process.runtime.name` - Fill in the value by the name of runtime.
+- `process.runtime.version` - Fill in the value of `System.Environment.Version`.
+- `process.runtime.description` - Fill in the values of `System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription`.
```

Comment:
`RuntimeInformation.FrameworkDescription` in .NET Framework only exists in v4.7.1+, but there are still some older [supported versions out there](https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet-framework).

On the other hand, the implementation currently [targets `net6.0`](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1449/files#diff-8134dd0c588313f1626ad7fbb5467122c50d45e6d0ae6e044659b6510531864eR4) from what I see. Which means it would not even report and run on .`NET Framework ...`.

Definitely only relevant for older apps - just wonder if this should be addressed, specifically 2 things:
- Would it make sense to call this out here, so people know they can't expect values like e.g. `.NET Framework 4.6.2`?
- Maybe add additional [targets to the implementation](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1449/files#diff-8134dd0c588313f1626ad7fbb5467122c50d45e6d0ae6e044659b6510531864eR4)? Specifically targeting .NET Framework as well.